roles/common/tasks/idle_deployment.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Scale down {{ deployment_type }} deployments
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: "{{ ansible_operator_meta.name }}-{{ item }}"
+        namespace: "{{ ansible_operator_meta.namespace }}"
+      spec:
+        replicas: 0
+  loop:
+    - api
+    - content
+    - web
+    - worker
+
+- name: Scale down Redis
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: "{{ ansible_operator_meta.name }}-redis"
+        namespace: "{{ ansible_operator_meta.namespace }}"
+      spec:
+        replicas: 0
```

Comment:
Since redis is a deployment kind and not statefulset then we could just add it to the loop on the previous task right ?

Reply:
Good idea, I just made that change and updated the PR.  That change works in local testing.